prisma/schema.prisma

```diff
@@ -314,3 +314,24 @@ model WorkTime {
   @@unique([weekDay, time, isDayOff])
   @@map("work_time")
 }
+
+model DonationDetails {
+  id                   String  @id @default(uuid()) @db.Uuid
+  donationEnabled      Boolean @default(true)
+  title                String  @unique @db.VarChar(50)
+  subtitle             String  @db.VarChar(50)
+  subtitleEnabled      Boolean @default(true)
```

Comment:
🟠 for all these `${fieldname}Enabled` - wouldn't be safe to assume if there is null | '' - it is disabled?

Reply:
Thanks for comment! Yep, those fields unavailable to be null. Allowing them to be null causes extra conditions on front-end(`${fieldname}Enabled &&...` ), which provides the same result with more code, so i found first variant to be better. Moreover on the admin panel, those fields are Boolean select , so they are always true or false.